src/api/index.ts

```diff
@@ -6,6 +6,7 @@ import ThemeEngines from '../generators/theme-engines';
 import {createOrUpdateDynamicTheme, removeDynamicTheme} from '../inject/dynamic-theme';
 import {collectCSS} from '../inject/dynamic-theme/css-collection';
 
+let isEnabled = false;
```

Comment:
Variable name is inconsistent.
also this will now collude with the `isEnabled` function.
I think `isDarkReaderActive` is a good variable or `isDarkReaderEnabled`.